src/apps/VirtualApp/components/MonitoringData.js

```diff
@@ -0,0 +1,111 @@
+// Monitoring library to track connection stats.
+import {
+  MONITORING_BACKEND,
+} from "../../../shared/consts";
+
+const ONE_SECOND_IN_MS = 1000;
+const STORE_INTERVAL = 60 * ONE_SECOND_IN_MS; // Store for one minute.
+
+export const MonitoringData = class {
```

Comment:
Why not regular function?

Reply:
This is a library class, not React component, nor hook. Just simple javascript library.

Reply:
Still class in not a native datatype in javascript, it's compiled to a function anyway, but to quite a complex one.

Reply:
Yes, you are correct. This is more of a style guide. Please let me know what is preferred.

Reply:
That's called Prototypal OO and it's supported directly even by IE6 ;)

A Class in JS:
function Animal(){  
    // Private property
    var alive=true;
    // Private method
    function fight(){ //... }   
    // Public method which can access private variables
    this.isAlive = function() { return alive; } 

    // Public property
    this.name = "Joe";
}

// Public method
Animal.prototype.play = function() { alert("Bow wow!"); }
// .. and so on

Now when you create it's object

var obj = new Animal();